pkgs/development/python-modules/ed25519-blake2b/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "ed25519-blake2b";
+  version = "1.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-0aHLkDLsMHzpW0HGGUQP1NP87MGPIkA1zH1tx6fY70A=";
+  };
+
```

Comment:
Missing pythonImportsCheck

Reply:
Also please rm __pycache__ in postPatch like the diff in https://github.com/Matoking/python-ed25519-blake2b/blob/master/test_ed25519_kat.py

Reply:
> Also please rm **pycache** in postPatch like the diff in https://github.com/Matoking/python-ed25519-blake2b/blob/master/test_ed25519_kat.py

These cached objects are not included in the PyPi distribution used here afaik.